scripts/crawl_expand.py

```diff
@@ -26,21 +26,23 @@
 where version is the newest version status
 """
 def expandPlaces(config, center, radius_km):
-    statusCache = db.child(venuesTable).child("status").get()
+    statusTable = db.child(venuesTable).child("status").get()
 
     # Fetch placeIDs to expand 
     location_table = db.child(locationsTable).get().val()
     placeIDs = geo.get_place_ids_in_radius(center, radius_km, location_table)
 
     for placeID in placeIDs:
-        placeStatus = statusCache.val()[placeID]
+        placeStatus = statusTable.val()[placeID]
         # Get a list of (src, version) pairs that could be updated 
         newSources = [src for src in config if src not in placeStatus or config[src] > placeStatus[src]]
         if not newSources:
             continue
-        request_handler.researchPlace(placeID, newSources, placeStatus["identifiers"])
-        print("done: %s" % placeID)
-        exit(0)
+        updatedSources = request_handler.researchPlace(placeID, newSources, placeStatus["identifiers"])
+        # Write updated sources to /status
+        placeStatus = db.child(venuesTable).child("status").child(placeID)
+        for source in newSources:
+            placeStatus.update({source: config[source] if source in updatedSources else 0})
```

Comment:
nit: should we use constants for the status values? e.g. 0 here could be `NOT_FETCHED`.